app/assets/javascripts/plate-map/overlay.js

```diff
@@ -14,41 +14,42 @@ var plateLayOutWidget = plateLayOutWidget || {};
         this.overLayTextContainer.text("Completion Percentage:");
         this.overLayContainer.append(this.overLayTextContainer);
         this.overLayButtonContainer = this._createElement("<div></div>").addClass("plate-setup-overlay-button-container");
+        this.overLayButtonContainer.attr("id", "plate-map-control-button-container-id");
         this.overLayContainer.append(this.overLayButtonContainer);
 
-        this.clearCriteriaButton = this._createElement("<button />").addClass("plate-setup-button");
+        this.clearCriteriaButton = this._createElement("<button />").addClass("plate-setup-button").attr('id', 'clear-id');
```

Comment:
In theory we could have multiple plate-maps per page, or the ids set here could clash with others on the page. If you really need to identify these buttons, (e.g. for hide/show with ebase) use `clear-button`, `copy-button`, etc. classes. Otherwise, don't use anything.